src/utils/xcode.rs

```diff
@@ -403,9 +403,12 @@ impl<'a> MayDetach<'a> {
             return Ok(false);
         }
 
+        let output_file = TempFile::create()?;
+
         println!("Continuing in background.");
+        println!("Output is redirected to {}", output_file.path().display());
```

Comment:
Perhaps it would make more sense to log this as a debug message, so we only print this line if the user specified the debug log level?

Reply:
I understand but I think there is so little information before going to the background that knowing the path is essential we should always log it.